t/BikeSharing/IndegoPHL.t

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+ddg_spice_test(
+    [qw( DDG::Spice::BikeSharing::IndegoPHL )],
+    'philly indego' => test_spice(
+        '/js/spice/bike_sharing/indego_phl/philly',
+        call_type => 'include',
+        caller => 'DDG::Spice::BikeSharing::IndegoPHL',
+    is_cached => 1,
```

Comment:
@moollaza I think the right thing to do is leave caching OFF (as we want up-to-date answers), and change the test not to expect a cached one